samples/routing/custom/Core_6/Shared/AutomaticRouting/RoutingInfoSubscriber.cs

```diff
@@ -129,7 +129,7 @@ public Task OnChanged(Entry entry)
         publishers.AddOrReplacePublishers("AutomaticRouting", newPublisherMap.Select(
             x => new PublisherTableEntry(x.Key, PublisherAddress.CreateFromEndpointName(x.Value))).ToList());
 
-        endpointInstances.AddOrReplaceInstances("AutomaticRouting", newInstanceMap.SelectMany(x => x.Value).ToList());
+        endpointInstances.AddOrReplaceInstances("AutomaticRouting", newInstanceMap.SelectMany(x => x.Value).Where(IsActive).ToList());
```

Comment:
this file already exists and seems to not directly belong to this sample? Are these changes necessary/related?